<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColumnizerWidget</class>
 <widget class="QWidget" name="ColumnizerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>799</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Colmunizer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="1" column="0">
      <widget class="QTableWidget" name="tableWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>tooltip</string>
       </property>
       <property name="toolTipDuration">
        <number>0</number>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string>whats this</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Preview</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QTableWidget" name="tablePreview">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="rowCount">
        <number>1</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row/>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbName">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="renameColumnizer">
           <property name="toolTip">
            <string>rename</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../../assets/default.qrc">
             <normaloff>:/assets/icons/icons8-edit-file-100.png</normaloff>:/assets/icons/icons8-edit-file-100.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="duplicateColumnizer">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>duplicate</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../../assets/default.qrc">
             <normaloff>:/assets/icons/icons8-new-copy-100.png</normaloff>:/assets/icons/icons8-new-copy-100.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteColumnizer">
           <property name="toolTip">
            <string>remove</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../../../assets/default.qrc">
             <normaloff>:/assets/icons/icons8-delete-document-100.png</normaloff>:/assets/icons/icons8-delete-document-100.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Column patterns</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="insertPatternBelow">
         <property name="toolTip">
          <string>insert row below</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../../assets/default.qrc">
           <normaloff>:/assets/icons/icons8-insert-row-100.png</normaloff>:/assets/icons/icons8-insert-row-100.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="insertPatternAbove">
         <property name="toolTip">
          <string>insert row above</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../../assets/default.qrc">
           <normaloff>:/assets/icons/icons8-insert-row-above-100.png</normaloff>:/assets/icons/icons8-insert-row-above-100.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deletePattern">
         <property name="toolTip">
          <string>delete row</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../../../assets/default.qrc">
           <normaloff>:/assets/icons/icons8-delete-row-100.png</normaloff>:/assets/icons/icons8-delete-row-100.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string>enter at least one whole logmessage here</string>
       </property>
       <property name="text">
        <string>Message to parse</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QTextEdit" name="editSubject">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string extracomment="Enter columns to parse">Enter columns to parse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../assets/default.qrc"/>
 </resources>
 <connections/>
</ui>
